src/components/spreadsheet-view/columns/common-column-definitions.ts

```diff
@@ -37,10 +37,18 @@ const updateAndPersistFilters = (colDef: ColumnDefinition, tab: string, api: Gri
     if (studyUuid) {
         const filter = filters?.find((f) => f.column === colDef.id);
         const columnDto = mapColDefToDto(colDef, filter);
-        updateSpreadsheetColumn(studyUuid, tab as UUID, colDef.uuid, columnDto);
+        updateSpreadsheetColumn(studyUuid, tab as UUID, colDef.uuid, columnDto).then();
```

Comment:
Is this needed ? Maybe a catch clause to display error would do the trick to prevent the lint issue instead